client_capabilities/assoc_req.py

```diff
@@ -105,7 +103,7 @@ def online_analysis():
     os.system('clear')
     print("#### Catching Association Request ####")
     sniff(iface=INTERFACE, monitor=True, filter="type mgt subtype assoc-req",
-          prn=assoc_req_parse(is_online=True), store=0)
+          prn=lambda x: assoc_req_parse(x, is_online=True), store=0)
```

Comment:
Here `prn` takes only one argument `x` due to lambda function declaration. Under the hood `assoc_req_parse` takes 2 arguments. Now `if` statement inside `assoc_req_parse` should work fine.